vignettes/_output.yml

```diff
@@ -0,0 +1,5 @@
+html_document:
```

Comment:
I wanted to exemplify the kind of setup where you have a vignette with no output format in the Rmd itself.